host-spec/c06-consensus.tm

```diff
@@ -351,25 +456,20 @@
   </definition>
 
   <\algorithm>
-    <label|algo-slot-time><name|Slot-Time>(<math|s>: the slot number of the
-    slot whose time needs to be determined)
+    <label|algo-slot-time><name|Median-Algorithm>(<math|\<frak-E\><rsub|j>>:
+    the current sync-epoch)
   <|algorithm>
     <\algorithmic>
       <\state>
         <math|T<rsub|s>\<leftarrow\><around*|{||}>>
       </state>
 
       <\state>
-        <math|B<rsub|d>\<leftarrow\>><name|Deepest-Leaf(<math|BT>)>
-      </state>
-
-      <\state>
-        <FOR-IN|<math|B<rsub|i>>|<name|SubChain>(<math|B<rsub|H<rsub|n><around*|(|B<rsub|d>|)>-SITL>>,
-        <math|B<rsub|d>>)>
+        <FOR-IN|<math|B<rsub|i>>|<math|\<frak-E\><rsub|j>><name|>>
```

Comment:
the blocks of the epoch are not defined anywhere.

Reply:
Indeed, the algorithm just ignores that fact. I will update it accordingly.

Reply:
Added this requirement to the text now.